src/index.ts

```diff
@@ -1,9 +0,0 @@
-/**
- * Example function that returns a greeting for the given name.
- *
- * @param name - The name to greet.
- * @returns The greeting.
- */
-export default function greeter(name: string): string {
-  return `Hello, ${name}!`;
-}
```

Comment:
This file is still being referenced as the entrypoint for the package in `package.json`. You'll probably want to re-export something from `ppom-storage` here, or update `package.json` to use an alternative entrypoint

Reply:
`index.js` with other exports will be added back in following PRs.